### ISSUE TITLE
/chatbomb- command to address chatbombing

### ISSUE THREAD

**Issue by JustWaveThings** [TheOdinProject/odin-bot-v2#451]
Complete the following REQUIRED checkboxes:
-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)

-   [x] The title of this issue follows the `command name: a brief description of request` format, e.g. `/help: add optional @user parameter`

The following checkbox is OPTIONAL:

-  [x] I would like to be assigned this issue to work on it
- Or I'd be happy to have someone else take it, as well. To me, this could be a good first command for someone interested in working with the bot -- maybe Bryan. I know they've expressed interest in the bot.

<hr>

**1. Description of the Feature Request:**

When users are aggressively chatbombing (like 3 or more messages in rapid succession in a row) I'd like a slash command that explains why it's bad, how to fix it going forward, and a link to the section of the rules that forbid it.

**2. Acceptance Criteria:**

- [ ] Optional ability to mention a user
- [ ] A slash command was created that addresses the 3 points mentioned in the description

**3. Additional Information:**

I know that we've moved away from using words like 'dogpiling', 'help vampire', and 'chatbombing' in general chat and the rules, so I'd need help picking a command name that didn't reference them.  I thought about /chatetiquette but that makes it seem optional. 

Also, it's not such a grave offense to require a thread, and I feel that it should be something that should be addressed when we see it in the public channel, much like crossposting is handled. 

Lastly, I searched the repo for 'chatbombing' and no hits so I assume that this was not previously discussed.

**Comment by JustWaveThings** [comment]
Yeah, I'm all for it being more positive than punitive.  But, I think this from the rules should be included in the command (rewritten if needed):
(from the same rule) 
> Avoid Posting multiple short messages in rapid succession because they are often incomplete sentences that make it hard for others to understand the full situation.

My reasoning for including this is that most chat bombers don't appear to be trying to 'ask a good question'. Their posts are *initially* short and they might assume they don't need to follow that convention.

I don't think we'd want to combine this command with one on assuming gender or other chat etiquette. I think they both should get their command so that the receiver doesn't feel like they are getting a laundry list thrown at them, and I can imagine a chatbomber responding to it that they didn't assume a gender and completely ignore the chatbombing part.

This could be a great command to have subcommands for.
- /chat - main command that just lists the subcommands 
- chatbomb - subcommand for chatbombers
- notallguys - subcommand for the gender
- nottop - subcommand  for telling people to go elsewhere when they ask for help on non-top issues /projects
- offtopic - subcommand for telling people to take the conversation to an off-topic channel
- showcase - subcommand for telling people looking for feedback on finished projects to post it there.
- techsupport - subcommand to direct people whose questions belong there

I might be going too far with this, feel free to reel me back in.

Also, is it a bad thing if someone feels just a bit called out when they engage in maladaptive behavior?  

A non-staff using the command isn't modding if they use the command, they are just helping the offending user recognize how things are done here. Just like someone referring a non-top question to another server.

**Comment by JustWaveThings** [comment]
Is this something we want to pursue still?

**Comment by 01zulfi** [comment]
@TheOdinProject/moderators what do we think

**Comment by JustWaveThings** [comment]
Closing. I now feel this command pushes Discord moderation in the wrong direction.
